lib/manageiq/password.rb

```diff
@@ -102,25 +89,6 @@ def self.try_encrypt(str)
       encrypted?(str) ? str : encrypt(str)
     end
 
-    # @returns [ver, enc]
-    def self.split(encrypted_str)
```

Comment:
did you say that removing this breaks a few callers?

you could alias this to `wrap` if you wanted to